Regular_Falsi/regu.c

```diff
@@ -0,0 +1,49 @@
+// Girlscript Summer Of Code
+// Goldmine problem  in java
+
+//In mathematics, the false position method or regula falsi is a very old method for solving an equation in one unknown,
+//that, in modified form, is still in use. In simple terms, the method is the trial and 
+//error technique of using test ("false") values for the variable and then adjusting the test value according
+//to the outcome. This is sometimes also referred to as "guess and check".
+//Versions of the method predate the advent of algebra and the use of equations.
+
+#include <stdio.h>
+#include<math.h>
+
+float
+f (float x)
+{
+  return ((x * x * x) - (2 * x) - 5);
```

Comment:
Throughout the code, follow 4-space indentation, 8-space indentation, and so on...